how-to/setup_new_users.rst

```diff
@@ -0,0 +1,92 @@
+JAAS: Setup New Users
+=====================
+
+This how-to describes how a new user can begin to use JAAS to deploy applications.
+
+Prerequisites
+-------------
+
+For this how-to you will need the following:
+
+- A running JAAS environment, see :doc:`our tutorial <../tutorial/deploy_jaas_microk8s>`.
+- A controller connected to JAAS, see :doc:`our how-to <./add_controller>`. This how-to assumes you have added an LXD controller.
+
+Login to JAAS
+-------------
+
+Login to JAAS for the first time:
+
+.. code:: bash
+
+    juju login test-jimm.localhost:443 -c jaas
+
+Running ``juju controllers`` will now show you the ``jaas`` controller.
+Commands like ``juju models`` will now work.
+
+Cloud Access
+------------
+
+Verify that you have access to a cloud.
+
+You can check this by running:
+
+.. code:: bash
+
+    juju clouds --controller jimm
+
+The response should resemble:
+
+.. code:: bash
+
+    Clouds available on the controller:
+    Cloud      Regions  Default    Type
+    localhost  1        localhost  lxd 
+
+If no clouds are available, an administrator may need to grant you
+``add-model`` permissions on this cloud. See the `juju grant-cloud <https://juju.is/docs/juju/juju-grant-cloud>`__ 
+command for details.
+
+Cloud Credentials
+-----------------
+
+Before we can create a model, your user requires a set of cloud `credentials <https://juju.is/docs/juju/credential>`__.
+
+View credentials available on your client:
+
+.. code:: bash
+
+    juju credentials --client
+
+The output should resemble the following
+
+.. code:: bash
+
+    Client Credentials:
+    Cloud      Credentials
+    localhost  localhost*
+    microk8s   microk8s*
+
+We will upload credentials for LXD - the `localhost` credentials.
+
+.. code:: bash
+
+    juju update-credentials localhost --controller jimm
+
+.. hint::
+
+    For clouds like Openstack or public clouds, use the `juju add-credential` command to interactively
+    add credentials to your client and controller.
+
+Credentials are now available on the controller for your user to deploy models on the LXD cloud.
```

Comment:
question: if credentials are only available on the client, won't they still be sent when adding a model?

Reply:
Do you mean, if I run `juju add-model` with no credentials on JIMM, it will automatically add/send the credenentials?

Reply:
I'll test it now but I don't think so.

Reply:
Tested it, I had the credential on my client but not on the jimm controller,
```
$ juju add-model foo localhost
ERROR failed to fetch cloud credentials localhost/jimm-test@canonical.com/localhost (not found)
```